src/Forms/TreeDropdownField.php

```diff
@@ -392,7 +393,12 @@ public function setShowSearch($bool)
      */
     public function getChildrenMethod()
     {
-        return $this->childrenMethod;
+        if ($this->childrenMethod) {
+            return $this->childrenMethod;
+        }
+
+        $sourceClass = $this->getSourceObject();
+        return $sourceClass::config()->get('default_children_method');
```

Comment:
Should this be calling `getHierarchyBaseClass()` and getting the config from that?

Reply:
I believe so, updated